org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/XMLSyntaxErrorCode.java

```diff
@@ -167,6 +167,10 @@ public static Range toLSPRange(XMLLocator location, XMLSyntaxErrorCode code, Obj
 			String tag = getString(arguments[0]);
 			return XMLPositionUtility.selectChildEndTag(tag, offset, document);
 		}
+		case SemicolonRequiredInReference: {

```

Comment:
I will trylike this:

```
case SemicolonRequiredInReference:
		case EntityNotDeclared: {
			EntityReferenceRange range = XMLPositionUtility.selectEntityReference(offset - 1, document, false);
			return range != null ? range.getRange() : null;
		}
```

Reply:
This doesn't include the ampersand, and just highlights the entity name. Should I include the ampersand in the highlight range?

Reply:
It should include it because when entity reference doesn't exist it highlight &. Are you sure that you are using the last master branch?

Reply:
I checked out my master branch, and updated to eclipse/lemminx master. I rebuilt the server and ran the VSCode extension. I get this as the error range: 
![ErrorRangeNoSemicolon](https://user-images.githubusercontent.com/22376627/83291936-419b3800-a1b7-11ea-9455-c4d0fff688e8.png)
The default range must have changed since this issue was opened.

Reply:
@datho7561 sorry with my mistake. Please keey EntityNotDeclared as today and add SemicolonRequiredInReference case like this:

```java
case SemicolonRequiredInReference: {
			EntityReferenceRange range = XMLPositionUtility.selectEntityReference(offset + 1, document, false);
			return range != null ? range.getRange() : null;
		}
```

Reply:
Thank you!